src/NetworkManager.cpp

```diff
@@ -72,6 +74,23 @@ NetworkManager::NetworkManager() : httpClient(true), downloadClient(true)
 		LUA->Release(L);
 	}
 
+	RageFile f;
+	if(f.Open(SpecialFiles::CA_BUNDLE_PATH))
+	{
+		RString data;
+		f.Read(data);
+		f.Close();
+
+		this->tlsOptions.caFile = data;
```

Comment:
What would happen if someone tried using the clients without a caFile? (just curious)

Reply:
Making TLS requests would fail.